src/main/java/com/avanza/gs/test/MirrorPu.java

```diff
@@ -56,7 +59,12 @@ public void run() throws IOException {
 	private void startContainers() throws IOException {
 		IntegratedProcessingUnitContainerProvider provider = new IntegratedProcessingUnitContainerProvider();
 		provider.setBeanLevelProperties(createBeanLevelProperties());
-		provider.addConfigLocation(puXmlPath);
+		if (puXmlPath != null) {
```

Comment:
I think it is valid; the `IntegratedProcessingUnitContainerProvider` would search for a `pu.xml` in `classpath*:/META-INF/spring/pu.xml`. This might not be desirable though, so in that sense it might not be valid.